### PR TITLE
Add TC meta-runners + documentations

### PR DESCRIPTION
**Pull request by ozonophore**

### CODE REVIEW

**Review by octopusden — APPROVED** [review]

**Review by pgorbachev — CHANGES_REQUESTED** [review]

File: teamcity.meta-runners/CalculateBuildVersion.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<meta-runner name="CalculateBuildVersion">
```
[the remark this answers is not in the document]
Comment (ozonophore) [reply teamcity.meta-runners/CalculateBuildVersion.xml]: Fixed

File: teamcity.meta-runners/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+There are some common steps in the configurations.
+They have been grouped by the configurations where they can be used.
+
+---
+
+### Compile & UT
+
+- [Read a latest version from tag](ReadVersionFromLatestTag.xml) - read the latest tag(by mask v(0-9).(0-9).(0-9)) from current Git repository and write it to the variable VERSION_TAG, also write a commit hash into CURRENT_COMMIT
+- [Calculate the build version](CalculateBuildVersion.xml) - parse a input variable VERSION_TAG with a special format (v{versionMajor}.{versionMinor}.{versionPatch}.{buildCounter}) and increment the patch version. The result is written into the PROJECT_VERSION with the format '$versionMajor.$versionMinor.$versionPatch' and into the variable buildNumber with the format $versionMajor.$versionMinor.$versionPatch-$buildCounter
```
[the remark this answers is not in the document]
Comment (ozonophore) [reply teamcity.meta-runners/README.md]: Fixed

**Review by pgorbachev — DISMISSED** [review]

**Review by pgorbachev — CHANGES_REQUESTED** [review]

File: teamcity.meta-runners/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- [Read a latest version from tag](ReadVersionFromLatestTag.xml) - read the latest tag(by mask v(0-9).(0-9).(0-9)) from current Git repository and write it to the variable VERSION_TAG, also write a commit hash into CURRENT_COMMIT
+- [Calculate octopus build version](CalculateBuildVersion.xml) - parse a input variable VERSION_TAG with a special format (v{versionMajor}.{versionMinor}.{versionPatch}.{buildCounter}) and increment the patch version. The result is written into the PROJECT_VERSION with the format '$versionMajor.$versionMinor.$versionPatch' and into the variable buildNumber with the format $versionMajor.$versionMinor.$versionPatch-$buildCounter
+    
+---
+# Pipe-line
+
+## Pipeline for Employee-service
+
+### 1. Compile && UT
```
Comment (pgorbachev) [inline teamcity.meta-runners/README.md:14]: Add "Specify RELENG_SKIP=true at project level"

**Comment by pgorbachev** [comment]
Specify also required changes for API Gateway (new identifier for keycloack client)

File: teamcity.meta-runners/README.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+There are some common steps in the configurations.
+They have been grouped by the configurations where they can be used.
+
+- [Read a latest version from tag](ReadVersionFromLatestTag.xml) - read the latest tag(by mask v(0-9).(0-9).(0-9)) from current Git repository and write it to the variable VERSION_TAG, also write a commit hash into CURRENT_COMMIT
+- [Calculate octopus build version](CalculateBuildVersion.xml) - parse a input variable VERSION_TAG with a special format (v{versionMajor}.{versionMinor}.{versionPatch}.{buildCounter}) and increment the patch version. The result is written into the PROJECT_VERSION with the format '$versionMajor.$versionMinor.$versionPatch' and into the variable buildNumber with the format $versionMajor.$versionMinor.$versionPatch-$buildCounter
+    
+---
+# Pipe-line
+
+## Pipeline for Employee-service
```
[the remark this answers is not in the document]
Comment (ozonophore) [reply teamcity.meta-runners/README.md:20]: fixed

File: teamcity.meta-runners/README.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+---
+# Pipe-line
+
+## Keycloak
+
+Add an ADMIN role to the user roles
+
+## Vault
+
+Add the new keys to ***'f1-config-server / f1-api-gateway-cloud-qa / f1-api-gateway-cloud-qa / spring.security.oauth2.client.registration.keycloak.client-secret'*** and ***'f1-config-server / f1-api-gateway-cloud-qa / f1-api-gateway-cloud-prod / spring.security.oauth2.client.registration.keycloak.client-secret'***
```
Comment (pgorbachev) [inline teamcity.meta-runners/README.md:18]: F1 здесь лишнее

**Review by pgorbachev — APPROVED** [review]

**Pull request merged**